src/engraving/dom/cmd.cpp

```diff
@@ -3010,13 +3010,16 @@ EngravingItem* Score::move(const String& cmd)
         }
     } else if (cmd == u"prev-chord" && cr) {
         // note input cursor
+        bool noteEntryPos = false;
         if (noteEntryMode() && m_is.segment()) {
             Measure* m = m_is.segment()->measure();
             Segment* s = m_is.segment()->prev1(SegmentType::ChordRest);
             track_idx_t track = m_is.track();
             for (; s; s = s->prev1(SegmentType::ChordRest)) {
                 if (s->element(track) || (s->measure() != m && s->rtick().isZero())) {
                     if (s->element(track)) {
```

Comment:
By the way, it would be great to simplify this a bit, while at it:
```suggestion
            for (; s; s = s->prev1(SegmentType::ChordRest)) {
                if (s->measure() != m && s->rtick().isZero()) {
                    break;
                }
                if (s->element(track)) {
```

Reply:
Now there is a closing `}` too much, sorry

Reply:
yeah, something went horribly wrong

Reply:
Like this now?